suites/pacific/cephadm/test-haproxy-deployment.yaml

```diff
@@ -187,3 +187,76 @@ tests:
           - "ingress.rgw.my-rgw"
       destroy-cluster: false
       abort-on-fail: true
+  - test:
+      name: Configure client
+      desc: Configure client on node5
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node5                       # client node
+        install_packages:
+          - ceph-common                   # install ceph common packages
+        copy_admin_keyring: true          # Copy admin keyring to node
+        store-keyring: true               # /etc/ceph/ceph.client.1.keyring
+        caps: # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: check-ceph-health on client node5
+      desc: Check for ceph health debug info
+      module: exec.py
+      config:
+        sudo: True
+        check_status: True
+        commands:
+          - "ceph status"
+  - test:
+      name: disable ha proxy service
+      desc: disable ha proxy service using systemctl commands
+      module: exec.py
+      config:
+        sudo: True
+        check_status: True
+        role: haproxy
+        service_name: haproxy
+        commands:
+          - "systemctl disable"
+  - test:
+      name: check-ceph-health and crash status on client node5
+      desc: Check for ceph health and crash status after ha proxy has been disabled
+      module: exec.py
+      config:
+        sudo: True
+        check_status: True
+        commands:
+          - "ceph status"
+          - "ceph health detail"
+          - "ceph crash stat"
+  - test:
+      name: enable ha proxy service
+      desc: enable ha proxy service using systemctl commands
+      module: exec.py
+      config:
+        sudo: True
+        check_status: True
+        role: haproxy
+        service_name: haproxy
+        commands:
+          - "systemctl enable"
+  - test:
+      name: check-ceph-health and crash status on client node5
+      desc: Check for ceph health and crash status after ha proxy has be reenabled
```

Comment:
**has been** there is spelling mistake Please correct that?

Reply:
Done.